From 11afcddc417f23c61f36ab918a8d1b2b0df2e5ed Mon Sep 17 00:00:00 2001
From: Jon Medhurst <tixy@linaro.org>
Date: Mon, 3 Mar 2014 16:29:47 +0000
Subject: [PATCH] ARM: vexpress: Disable PSCI node in TC2 device-tree

The standard TC2 firmware doesn't support PSCI.

Signed-off-by: Jon Medhurst <tixy@linaro.org>
(cherry picked from commit da6baa2efbaf08915eea798487b23eaf2626aab7)
Signed-off-by: Alex Shi <alex.shi@linaro.org>
---
 arch/arm/boot/dts/vexpress-v2p-ca15_a7.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/vexpress-v2p-ca15_a7.dts b/arch/arm/boot/dts/vexpress-v2p-ca15_a7.dts
index 9577062..df060c1 100644
--- a/arch/arm/boot/dts/vexpress-v2p-ca15_a7.dts
+++ b/arch/arm/boot/dts/vexpress-v2p-ca15_a7.dts
@@ -178,6 +178,9 @@
 		clock-output-names = "oscclk6a";
 	};
 
+/* PSCI requires support from firmware and is not present in the normal TC2
+ * distribution, so this node is commented out by default...
+
 	psci {
 		compatible      = "arm,psci";
 		method          = "smc";
@@ -186,6 +189,7 @@
 		cpu_on          = <0x80100003>;
 		migrate         = <0x80100004>;
 	};
+*/
 
 	dcc {
 		compatible = "arm,vexpress,config-bus";
-- 
1.8.0

